licenses.yaml

```diff
@@ -1038,11 +1004,26 @@ notice: |
 
 ---
 
+name: Netty
+license_category: binary
+module: java-core
+license_name: Apache License version 2.0
+version: 4.1.30.Final
+libraries:
+  - io.netty: netty-buffer
+  - io.netty: netty-codec
+  - io.netty: netty-codec-http
+  - io.netty: netty-common
+  - io.netty: netty-handler
+  - io.netty: netty-transport
```

Comment:
There is already an entry for this version of Netty, https://github.com/apache/incubator-druid/blob/557f2cc0395c6f9b1082ee8a7742b70664dba45a/licenses.yaml#L761. Additionally, Netty has a `NOTICE`, I think we need to add the text of it from this version similar to how 4.1.29 is.

Reply:
Actually, are there still 4.1.29 jars in distribution package?

Reply:
There's a mix of netty4 versions.

Before my changes:
```
netty-all-4.1.30.Final.jar
netty-buffer-4.1.29.Final.jar
netty-codec-4.1.29.Final.jar
netty-codec-dns-4.1.29.Final.jar
netty-codec-http-4.1.29.Final.jar
netty-codec-socks-4.1.29.Final.jar
netty-common-4.1.29.Final.jar
netty-handler-4.1.29.Final.jar
netty-handler-proxy-4.1.29.Final.jar
netty-resolver-4.1.29.Final.jar
netty-resolver-dns-4.1.29.Final.jar
netty-transport-4.1.29.Final.jar
netty-transport-native-epoll-4.1.29.Final-linux-x86_64.jar
netty-transport-native-unix-common-4.1.29.Final.jar
```

After my changes:
```
netty-buffer-4.1.30.Final.jar
netty-codec-4.1.30.Final.jar
netty-codec-dns-4.1.29.Final.jar
netty-codec-http-4.1.30.Final.jar
netty-codec-socks-4.1.29.Final.jar
netty-common-4.1.30.Final.jar
netty-handler-4.1.30.Final.jar
netty-handler-proxy-4.1.29.Final.jar
netty-resolver-4.1.29.Final.jar
netty-resolver-dns-4.1.29.Final.jar
netty-transport-4.1.30.Final.jar
netty-transport-native-epoll-4.1.29.Final-linux-x86_64.jar
netty-transport-native-unix-common-4.1.29.Final.jar
```

I'll fix it so the same version gets used.

Reply:
Got it, thanks. Also it looks like notice between 2 versions is identical, so maybe is ok to not have it twice.

Reply:
Still should consolidate the two `4.1.30.Final` entries i think though